### ISSUE TITLE
extraPipPackages not getting installed after migrating to 8.0.8

### ISSUE THREAD

**Issue by griseau** [airflow-helm/charts#163]
**What is the bug?**

I tried to upgrade to 8.0.8 to fix #124, but it looks like extraPipPackages are not installed anymore. 
I was previously using 8.0.2 so not sure if this is due to 8.0.8, but as @thesuperzapper added a new init container I'm guessing the init with install-pip-packages doesn't work well ? 

**What are your Helm values?**
```yaml
airflow.extraPipPackages: 
    - "SQLAlchemy==1.3.23"
    - "apache-airflow-providers-databricks==1.0.1"
    - "azure-keyvault-secrets==4.2.0"
    - "azure-identity==1.5.0"
```

**What is your Kubernetes Version?**:

```console
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.7", GitCommit:"1dd5338295409edcfff11505e7bb246f0d325d15", GitTreeState:"clean", BuildDate:"2021-01-13T13:23:52Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.14", GitCommit:"5de7fd1f9555368a86eb0f8f664dc58055c17269", GitTreeState:"clean", BuildDate:"2021-01-18T09:31:01Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

**What is your Helm version?**:

```console
version.BuildInfo{Version:"v3.5.2", GitCommit:"167aac70832d3a384f65f9745335e9fb40169dc2", GitTreeState:"dirty", GoVersion:"go1.16rc1"}
```

**Comment by thesuperzapper** [comment]
@griseau is this happening for all pod types (deployment, etc) or just one?

**Comment by thesuperzapper** [comment]
@griseau @simmartynas could you please post logs from the pod, because I just tried with `extraPipPackages` and `pandas` and it seems to work?

**Comment by griseau** [comment]
@thesuperzapper, wasn't working the first time I tried, I did a rollback, upgrade again and that time it worked. I'll try on a new clean env to see if I can reproduce and share the logs.

**Comment by simmartynas** [comment]
Currently I have problem with `gsutil` pip package 

Here are logs that this package was successfully installed in worker pod
![image](https://user-images.githubusercontent.com/16124418/115353759-91189700-a1c1-11eb-85c3-2a092c29db08.png)

But running bash command with airflow says
```[2021-04-20 03:44:50,040] {bash.py:173} INFO - bash: line 1: gsutil: command not found```

Checked inside worker pod also says it's missing:
![image](https://user-images.githubusercontent.com/16124418/115354007-d6d55f80-a1c1-11eb-9be1-2d442fbb55b8.png)

Trying manually uninstalling and installing this package, seems to help
![image](https://user-images.githubusercontent.com/16124418/115354424-48151280-a1c2-11eb-9232-d81a9b4791d6.png)

**Comment by thesuperzapper** [comment]
@simmartynas why would installing a python package bring something into your bash PATH?

**Comment by simmartynas** [comment]
@thesuperzapper It's used as command line tool. It worked with `7.14.0` version.

**Comment by thesuperzapper** [comment]
@simmartynas this is because we no longer prepend the `/home/airflow/.local/bin` to PATH (in `8.X.X`), but furthermore, even if we did it wouldn't have the same effect as in `7.X.X`, as we now install into `/opt/python/site-packages`.

I will have to think about how we should solve this, can you raise a new issue so we can discuss it without pouting this one?

**Comment by griseau** [comment]
Have another issue #165, so I can't reproduce at the moment but I'll let you know asap @thesuperzapper

**Comment by thesuperzapper** [comment]
@griseau I believe this issue is the same as https://github.com/airflow-helm/charts/issues/168

Lets move discussion there.

**Comment by thesuperzapper** [comment]
Hi all, this should be resolved after version `8.0.9` of the chart.
